packages/datatrak-web-server/src/app/createApp.ts

```diff
@@ -18,8 +18,8 @@ import {
   UserRequest,
   SurveysRoute,
   SurveysRequest,
-  SurveyResponsesRequest,
-  SurveyResponsesRoute,
+  SubmissionHistoryRequest,
```

Comment:
hmmm I don't think it's necessary to have the api endpoint name match the name of the page section. I think it's more desirable for the server to know less about the front-end where possible and return a resource for the frontend to consume. So I would personally prefer to keep the name of the route as SurveyResponsesRoute.